docs/trace/building-your-own-exporter/README.md

```diff
@@ -0,0 +1,41 @@
+# Building your own Exporter
+
+* To export telemetry to a specific destination, custom exporters must be
+  written.
+* Exporters should inherit from `ActivityExporter` and implement `ExportAsync`
+  and `ShutdownAsync` methods. `ActivityExporter` is part of the [OpenTelemetry
+  Package](https://www.nuget.org/packages/opentelemetry).
+* Depending on user's choice and load on the application, `ExportAsync` may get
+  called with zero or more activities.
+* Exporters will only receive sampled-in and ended activities.
+* Exporters must not throw.
+* Exporters should not modify activities they receive (the same activity may be
+  exported again by different exporter).
+* Any retry logic that is required by the exporter is the responsibility of the
```

Comment:
@reyang Does this look correct now? 

from spec:
_Any retry logic that is required by the exporter is the responsibility of the exporter. The default SDK SHOULD NOT implement retry logic, as the required logic is likely to depend heavily on the specific protocol and backend the spans are being sent to._
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#span-exporter